app/helpers/application_helper/toolbar_builder.rb

```diff
@@ -865,6 +865,11 @@ def build_toolbar_hide_button(id)
       end
     when "OrchestrationTemplate", "OrchestrationTemplateCfn", "OrchestrationTemplateHot", "OrchestrationTemplateAzure"
       return true unless role_allows(:feature => id)
+    when "ManageIQ::Providers::AnsibleTower::ConfigurationManager::ConfiguredSystem", "ManageIQ::Providers::Foreman::ConfigurationManager::ConfiguredSystem"
```

Comment:
It would be nice to just use `when ConfiguredSystem` rather than naming all subclasses, but with the size of this method, that is clearly out of scope for this PR.